source/dub/compilers/utils.d

```diff
@@ -268,78 +268,103 @@ private enum probeEndMark = "__dub_probe_end__";
 */
 NativePath generatePlatformProbeFile()
 {
+
 	import dub.internal.vibecompat.core.file;
-	import dub.internal.vibecompat.data.json;
 	import dub.internal.utils;
 	import std.string : format;
 
+	enum moduleInfo = q{
+		module object;
+		alias string = const(char)[];
+	};
+
 	// try to not use phobos in the probe to avoid long import times
```

Comment:
I would also modify the comment so the next person knows the platform probe should not rely on the runtime to make your use case possible.